src/javascript/app/pages/user/account/settings/self_exclusion.js

```diff
@@ -268,10 +279,16 @@ const SelfExclusion = (() => {
                 });
             }
 
-            $('#gamstop_info_bottom').setVisibility(is_gamstop_client && date);
+            showWarning(timeout_val || exclude_until_val);
         });
     };
 
+    const showWarning = (is_enable = true) => {
```

Comment:
can we use `is_enabled` to be grammatically correct?
```suggestion
    const showWarning = (is_enabled = true) => {
```